rubix-tests/src/test/robotframework/shared/docker/create_docker_compose.py

```diff
@@ -0,0 +1,46 @@
+#!/usr/bin/env python
+import sys
+
+master_str = """version: '3.7'
+networks:
+  default:
+    external:
+      name: network-rubix-build
+services:
+  rubix-master:
+    build:
+      context: .
+      args:
+        is_master: \"true\"
+    volumes:
+      - /tmp/rubix/tests:/tmp/rubix/tests
+      - /tmp/rubix/jars:/usr/lib/rubix/lib
+    networks:
+      default:
+        ipv4_address: 172.18.8.0\n"""
+
+
+worker_str = """:  
```

Comment:
Is it possible to use a formatted string here for filling in the service name and IP address?